Singularity

```diff
@@ -0,0 +1,44 @@
+Bootstrap: docker
+From: ubuntu:devel
+# Requires: Singularity 2.3+
+
+%help
+Welcome to the openPMD-api container.
+This container contains a pre-installed openPMD-api library.
+It supports serial and MPI parallel I/O with HDF5 and ADIOS1 and provides python bindings.
+
+%setup
+    mkdir -p ${SINGULARITY_ROOTFS}/opt/openpmd-api
+
+%files
+    ./* /opt/openpmd-api
+
+%post
+apt-get update && \
+apt-get install -y --no-install-recommends \
+    cmake \
+    ipython3 \
+    pybind11-dev \
+    libhdf5-openmpi-dev \
+    libopenmpi-dev && \
+rm -rf /var/lib/apt/lists/*
+# libadios-openmpi-dev
```

Comment:
`libadios` is currently only provided as non-mpi variant in Debian/Ubuntu. Opened upstream request in https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=900804